clients.rst

```diff
@@ -1,11 +1,17 @@
 Clients & Adapters
 ==================
 
-There are clients implementing one of the HTTPlug interfaces directly,
-and adapter packages that implement the interface and forward the calls to HTTP clients not implementing the interface.
+There are two types of libraries you can use to send HTTP messages; clients and adapters. A client implements the
+``HttpClient`` and/or the ``HttpAsyncClient`` interfaces directly. A client adapter is a class implementing the
+interface and forward the calls to a HTTP client not implementing the interface. (See `Adapter pattern`_ on Wikipedia).
```

Comment:
s/a HTTP/an HTTP/ (english grammar is about how its pronounced. H sounds like ahsh or something, so the a needs the n to link.